Traceback (most recent call last):
  File "/home/hawkwood/.local/bin/molecule", line 8, in <module>
    sys.exit(main())
  File "/home/hawkwood/.local/lib/python3.8/site-packages/click/core.py", line 1137, in __call__
    return self.main(*args, **kwargs)
  File "/home/hawkwood/.local/lib/python3.8/site-packages/click/core.py", line 1062, in main
    rv = self.invoke(ctx)
  File "/home/hawkwood/.local/lib/python3.8/site-packages/click/core.py", line 1668, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/hawkwood/.local/lib/python3.8/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/hawkwood/.local/lib/python3.8/site-packages/click/core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "/home/hawkwood/.local/lib/python3.8/site-packages/click/decorators.py", line 26, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/home/hawkwood/.local/lib/python3.8/site-packages/molecule/command/test.py", line 159, in test
    base.execute_cmdline_scenarios(scenario_name, args, command_args, ansible_args)
  File "/home/hawkwood/.local/lib/python3.8/site-packages/molecule/command/base.py", line 97, in execute_cmdline_scenarios
    get_configs(args, command_args, ansible_args, glob_str), scenario_name
  File "/home/hawkwood/.local/lib/python3.8/site-packages/molecule/command/base.py", line 184, in get_configs
    configs = [
  File "/home/hawkwood/.local/lib/python3.8/site-packages/molecule/command/base.py", line 185, in <listcomp>
    config.Config(
  File "/home/hawkwood/.local/lib/python3.8/site-packages/molecule/config.py", line 63, in __call__
    obj = type.__call__(cls, *args, **kwargs)
  File "/home/hawkwood/.local/lib/python3.8/site-packages/molecule/config.py", line 109, in __init__
    self.runtime = Runtime(isolated=True)
  File "/home/hawkwood/.local/lib/python3.8/site-packages/ansible_compat/runtime.py", line 78, in __init__
    self.config = AnsibleConfig()
  File "/home/hawkwood/.local/lib/python3.8/site-packages/ansible_compat/config.py", line 431, in __init__
    config_dump = subprocess.check_output(
  File "/usr/lib/python3.8/subprocess.py", line 415, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/usr/lib/python3.8/subprocess.py", line 493, in run
    with Popen(*popenargs, **kwargs) as process:
  File "/usr/lib/python3.8/subprocess.py", line 858, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/lib/python3.8/subprocess.py", line 1704, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'ansible-config'
